snapcraft/tests/sources/test_rpm.py

```diff
@@ -61,3 +62,9 @@ def test_extract_and_keep_rpmfile(self):
 
         test_output_files = ['test.txt', rpm_file_name]
         self.assertCountEqual(os.listdir(dest_dir), test_output_files)
+
+    def test_has_source_handler_entry_on_linux(self):
+        if sys.platform == 'linux':
+            self.assertTrue(sources._source_handler['rpm'] is sources.Rpm)
+        else:
+            self.assertRaises(KeyError, sources._source_handler['rpm'])
```

Comment:
No, I meant for both to be like the rpm case. Opened atom to find not all files had been saved though!